flow-typed/npm/react-datepicker_v2.x.x.js

```diff
@@ -0,0 +1,170 @@
+// flow-typed version based on https://github.com/Hacker0x01/react-datepicker/blob/master/docs/datepicker.md
+
+import React from 'react';
+
+declare module 'react-datepicker' {
+
+  declare type LocaleData = {
```

Comment:
@AlbertCarreras could we add a link to https://github.com/date-fns/date-fns/blob/81ab18785146405ca2ae28710cdfbb13a294ec50/src/locale/af/index.js.flow

Reply:
fixed